test/uk/gov/hmrc/servicedependencies/model/ApiSlugInfoFormatsSpec.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.servicedependencies.model
+
+import java.time.LocalDateTime
+
+import org.scalatest.FlatSpec
+import play.api.libs.json.Json
+
+class ApiSlugInfoFormatsSpec extends FlatSpec {
+
+  "api json format" should "handle missing javainfo sections " in {
```

Comment:
Looks like this test might not be complete. I presume `jsonWithoutJavaInfo` should be used in this test.

Reply:
originally was planning to support the old and new format of java version info. After thinking about it seemed easier just to migrate the data in the db to the new format, either in place or in artefact processor then backfilling. I'll remove this test since its unneeded